### ISSUE TITLE
Need to add Trim feature.

### ISSUE THREAD

**Issue by briankurzhal** [shaise/FreeCAD_SheetMetal#86]
Need to add Trim feature in order to create quadrilaterals.

**Comment by luzpaz** [comment]
@shaise add 'enhancement' label ?

**Comment by shaise** [comment]
added

**Comment by luzpaz** [comment]
@briankurzhal do you mind posting this Feature Request on the dedicated SheetMetal thread on the FreeCAD subforum to generate discussion and possibly integration of this enhancement in to the workbench? 

Forum thread: https://forum.freecadweb.org/viewtopic.php?f=10&t=11303

**Comment by briankurzhal** [comment]
Good idea, luzpaz, and thank you.  It appears that shaise might have added my Feature Request to the dedicated Sheetmetal thread.  If so, then many thanks to you both.  Much appreciated.
